director/origin_api.go

```diff
@@ -264,6 +270,6 @@ func GetRegistryIssuerURL(prefix string) (string, error) {
 	if err != nil {
 		return "", err
 	}
-	namespace_url.Path = path.Join(namespace_url.Path, "api", "v1.0", "registry", prefix, ".well-known", "issuer.jwks")
+	namespace_url.Path = path.Join(namespace_url.Path, "api", "v2.0", "registry", "metadata", prefix, ".well-known", "issuer.jwks")
```

Comment:
I'm worried that on Windows this will use the wrong path separator, because `path.Join` is os-specific. Can you use the `url.JoinPath` method here?